cmake/modules/SetUpMacOS.cmake

```diff
@@ -85,6 +85,17 @@ if (CMAKE_SYSTEM_NAME MATCHES Darwin)
 
      set(CMAKE_C_LINK_FLAGS "${CMAKE_C_LINK_FLAGS} -bind_at_load -m64")
      set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} -bind_at_load -m64")
+     
+     if(asan)
+       # See also core/sanitizer/Readme.md for what's happening.
+
```

Comment:
Empty line

Reply:
Is that a problem? Thought it would be nice to separate the info on what's happening from the attempt to do it right, which fails to do the clang bug.

Reply:
I am not picky, just noticed, thanks  for explanation :)